src/main/java/lotto/controller/LottoStore.java

```diff
@@ -1,86 +1,92 @@
 package lotto.controller;
 
-import java.util.EnumMap;
-import java.util.Map.Entry;
+import java.util.List;
 import java.util.Scanner;
 import lotto.domain.LottoAnnouncement;
-import lotto.domain.LottoRank;
+import lotto.domain.LottoProfitRate;
+import lotto.domain.LottoResult;
 import lotto.domain.Lottos;
 import lotto.domain.Money;
-import lotto.controller.generator.LottoAutoGenerator;
-import lotto.viewer.AnnouncementInputView;
-import lotto.viewer.MoneyInputView;
+import lotto.domain.Piece;
+import lotto.domain.generator.LottoAutoNumberGenerator;
+import lotto.domain.generator.LottoManualNumberGenerator;
+import lotto.exception.LottoAnnouncementException;
+import lotto.exception.LottoException;
+import lotto.exception.MoneyException;
+import lotto.exception.PieceException;
+import lotto.viewer.InputView;
 import lotto.viewer.OutputView;
 
 public class LottoStore {
 
-    private final static int LOTTO_PRICE = 1000;
-    private static final int DECIMAL_TRIM_NUMERATOR = 100;
-    private static final double DECIMAL_TRIM_DENOMINATOR = 100.00;
-
-    private final MoneyInputView moneyInputView;
-    private final AnnouncementInputView announcementInputView;
+    private final InputView inputView;
     private final OutputView outputView;
 
     public LottoStore() {
         Scanner scanner = new Scanner(System.in);
-        moneyInputView = new MoneyInputView(scanner);
-        announcementInputView = new AnnouncementInputView(scanner);
+        inputView = new InputView(scanner);
         outputView = new OutputView();
     }
 
     public void process() {
         Lottos lottos = buyLotto();
         LottoAnnouncement lottoAnnouncement = receiveValidLottoAnnouncement();
-        EnumMap<LottoRank, Integer> lottoResultStatistics = lottos.getStatistics(lottoAnnouncement);
-        printLottoResult(lottoResultStatistics, lottos);
+        LottoResult lottoResult = new LottoResult(lottoAnnouncement, lottos);
+        LottoProfitRate lottoProfitRate = new LottoProfitRate(lottoResult, lottos.getSize());
```

Comment:
```suggestion
        LottoProfitRate lottoProfitRate = lottoResult.getProfitRate();
```

꼭 데이터를 가져와서 생성할 필요는 없습니다!
`메시지를 던져라`라는 의미에 대해 고민해보아요